modules/ztp-sno-accelerated-ztp.adoc

```diff
@@ -0,0 +1,76 @@
+// Module included in the following assemblies:
+//
+// * edge_computing/ztp-deploying-far-edge-sites.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="ztp-sno-accelerated-ztp_{context}"]
+= Accelerated provisioning of {ztp}
+
+:FeatureName: Accelerated provisioning of {ztp}
+include::snippets/technology-preview.adoc[]
+
+You can reduce the time taken for installation by using accelerated provisioning of {ztp} for {sno}. 
+Accelerated ZTP speeds up installation by applying Day 2 manifests derived from policies at an earlier stage. 
+
+[IMPORTANT]
+====
+Accelerated provisioning of {ztp} is supported only when installing {sno} with Assisted Installer. Otherwise this installation method will fail.
+====
+
+== Activating accelerated ZTP
```

Comment:
Each heading needs to have a unique ID per our [docs guidelines](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#anchoring-titles-and-section-headings):
```suggestion
[id="activating-accelerated-ztp_{context}"]
== Activating accelerated ZTP
```

Up to you what the actual ID text is, this is just one suggestion

Reply:
good suggestion. Thanks.